examples/pytorch/nlp/huggingface_models/language-modeling/pruning/magnitude/README.md

```diff
@@ -0,0 +1,90 @@
+Step-by-Step
+============
+
+# single GPU
+
+```
+bash run.sh
+```
+
+# multi GPU
+
+we use `accelerate` and `deepspeed ZeRO Stage-2` to conduct weight magnitude pruning
+
+### Accelerate DeepSpeed Plugin
```

Comment:
where to specify the number of GPU cards?

Reply:
accelerate config commands will ask you to set the gpu cards